app/src/main/java/appricottsoftware/clarity/HomeActivity.java

```diff
@@ -73,14 +85,31 @@ public void onBackPressed() {
         moveTaskToBack(true);
     }
 
+    private void setUpPlayer() {
+        FragmentManager fragmentManager = getSupportFragmentManager();
+        fragmentManager.beginTransaction()
+                .replace(R.id.fl_home_activity_player, new PlayerFragment())
```

Comment:
Make sure to make the beginning "." aligned with the others. 

Therefore it should be

fragmentManager.beginTransaction() 
                              .replace(....)

Reply:
Indenting the 2nd line makes the code more readable. It is part of the previous line of code.